WalletWasabi/Gma/QrCodeNet/Encoding/DataEncodation/ECISet.cs

```diff
@@ -211,8 +211,7 @@ internal BitList GetECIHeader(string encodingName)
 			int eciAssignmentByte = NumOfCodewords(eciValue);
 			//Number of bits = Num codewords indicator + codeword value = Number of codewords * 8
 			//Chapter 6.4.2.1 ECI Designator ISOIEC 18004:2006 Page 24
-			int eciAssignmentBits = 0;
```

Comment:
Why do you remove the ` = 0` here?

Reply:
Because the default value for an `int` is 0.